versionedRouter.js

```diff
@@ -106,6 +111,10 @@ async function handleDest(ctx, version, destination) {
   });
   const respList = [];
   const executeStartTime = new Date();
+  // destination definition is going to be same for any kind of event
+  if (!isCdkDestination(events[0])) {
+    destHandler = getDestHandler(version, destination);
+  }
```

Comment:
There is a bug on Ln 127. It should have been `parsedEvent` rather than `event`.
CC @sanpj2292

Reply:
Changed it, can you check now ?